lib/user.js

```diff
@@ -31,9 +33,23 @@ const instanceMethods = {
         });
     },
 
-    callFunction(name, args) {
+    callFunction(name, args, service = undefined) {
```

Comment:
Added the optional `service` param to support `remoteMongoClient()`. Also now using `cleanArgs()` preprocessing similar taken from https://github.com/realm/realm-js/blob/11f690217253a095cf605779d006f0be9f7cb6a1/packages/realm-web/src/FunctionsFactory.ts#L51-L68